### PR TITLE
2019.9: Setting build time variables: init

### PR DESCRIPTION
**Pull request by oyvindsk**
Is this understandable? :D Or should I explain a little more?

### CODE REVIEW

**Comment by oyvindsk** [comment]
https://docs.google.com/presentation/d/1YGkPflHXeo4P3t4Hhb3HvASoTxqqzzwEbaj-8wMnEPA/

File: 2019.10/setting-build-time-variables/README.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+What:
+Set variables from the "outside" when compiling (go buidl, install, run etc..)
+
+Why:
+Some information is too dynamic to hardcode in the source code, but, at the same time, not something we can figure out at runtime.
+
+The version of a program, for example, is a pain to have in in the source code beacuse it's very easy to forget to update it every time you compile a new version.
```
Comment (xamebax) [inline 2019.10/setting-build-time-variables/README.txt]: There's a typo: "beacuse" instead of "because" :)

**Comment by xamebax** [comment]
I think it looks good. :+1: I just noticed a typo (separate comment).

File: 2019.10/setting-build-time-variables/README.txt (hunk 1 above)
Comment (oyvindsk) [reply 2019.10/setting-build-time-variables/README.txt]: thanks :+1:

**Pull request merged**